src/backend/user.js

```diff
@@ -0,0 +1,24 @@
+module.exports = class User {
+    constructor(options) {
+        this.id = options.id || 'no id';
+        this.name = options.name || 'no name'; // test fail if empty
+        this.joined = options.joined || 'joined For';
+
+    }
+
+    display() {
+        return {
+            id: this.id,
+            username: this.name,
+            displayName: this.name.charAt(0).toUpperCase() + this.name.slice(1),
+            twitter: '@' + this.name,
+            memberFor: (Date.now() - this.joined) + 'miliseconds'
+        };
+    }
+
+    prepareToInsert(callback) {
+        let keys = ['name', 'joined']; // propertisy
+        callback( keys, this);
```

Comment:
Nie rozumiem dlaczego ta metoda jest w ogóle potrzebna. 
I nie zrozum mnie źle, widzę gdzie jest użyta i jak, ale zupełnie nie rozumiem po co używać metody z callbackiem do zrobienia czegoś synchronicznego, co w dodatku ogranicza się do zwrócenia dwóch hardcodowanych wartości. Bo do this już był dostęþ przecież w miejscu, gdzie metoda jest wołana.